### ISSUE TITLE
Address hreflang issues

### ISSUE THREAD

**Issue by nickpistone** [GSA/px-benefit-finder#1032]
### Description

ahrefs has the same 3 hreflang issues for each of the BF pages. They seem to be all intertwined so I just created one ticket to address all of them. I have listed out the three issues and why and how to fix them below.
**Missing reciprocal hreflang (no return-tag)**

Issue details
Confirmation (return) links are missing for the pages declared in hreflang annotations.

If page A links to page B in hreflang annotations, page B must link to page A in return. If this is not the case for all pages that use hreflang annotations, those annotations may be ignored or misinterpreted correctly.

This issue can also be triggered by misused "x-default" hreflang. Whenever the optional "x-default" hreflang value is used, all pages of the hreflang group must reference the same "default" page.

How to fix
To fix the issue, make sure all the page versions translated or targeted to users in a certain region have the same set of hreflang annotations which includes URLs to all the page versions. This will always provide return links between the alternate pages.

[Learn more](https://help.ahrefs.com/site-audit/missing-reciprocal-hreflang-no-return-tag-error-in-site-audit)

**Self-reference hreflang annotation missing**

Issue details
It's a good practice for each language version of a page to list itself in addition to all other language versions. While it is not a strict requirement for Google, hreflang annotations without the self-referencing link may be ignored or misinterpreted by other search engines.

How to fix
Review all the pages with missing self-referencing hreflang annotations listed in this report. You can see all hreflang links specified for a given page in the table.

Make the necessary changes, so that each of these pages has a self-referencing hreflang annotation.

For example, a Spanish version of your page must have rel="alternate" hreflang="es" annotation with a link to itself.

**X-default hreflang annotation missing**

Issue details
Pages with no x-default hreflang annotation.

A set of hreflang annotations on the multiple versions of your page translated or targeted to users in a certain region should include an x-default hreflang attribute.

x-default will be used for any other language (or language and location) not specified in a set of hreflang annotations.

How to fix
Make sure each page on your website has an x-default hreflang attribute that points to a page not specific to one language or region.

See Google's guidelines on hreflang implementation [here](https://support.google.com/webmasters/answer/189077?hl=en).

### User Story

#1027

**Comment by scottqueen-bixal** [comment]
This is inherited from a USAGov template partial here https://github.com/GSA/px-benefit-finder/blob/dev/usagov_benefit_finder/modules/usagov_benefit_finder_app/usagov_benefit_finder_page/templates/page--benefit-finder-life-event.html.twig#L55

is this an issue on `/benefit-finder` landing page?

**Comment by nickpistone** [comment]
@scottqueen-bixal yes this issue (3 related issues) is the same for all 8 of our pages including the English and Spanish landing pages. This is the one I understand the least.

**Comment by scottqueen-bixal** [comment]
Based on your response, I think this is an issue that needs to be brought to the attention of usagov since it is also an issue on the landing page, then it likely is an issue accross the site. 

Recomend sharing this finding with usagov to see if they are willing to confirm/address.

**Comment by nickpistone** [comment]
I have shared with David Kaufmann to see what he thinks.

**Comment by nickpistone** [comment]
Leaving this up to David Kaufmann is he wants to address on his end.
